<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QWidget" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>736</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainInterface</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>3</y>
     <width>390</width>
     <height>732</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid rgb(100, 169,139);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="head" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>410</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>388</width>
      <height>150</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#head{
border:none;
background-color: qlineargradient(x1:1,y1:0,x2:0,y2:0,stop:0  #29bdd9,stop:1 #276ace);
}
QLabel{
color:white;
}</string>
    </property>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>RhCsBe</string>
     </property>
    </widget>
    <widget class="QLabel" name="signature">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>211</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>编辑个性签名</string>
     </property>
    </widget>
    <widget class="QToolButton" name="min_btn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="最小化"/>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:hover
{
	background-color:rgba(200, 200, 200,85);
}
QToolButton:pressed
{
	background-color:rgba(220,220,220,85);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton:hover 
{
	background-color:rgb(255, 0, 0);
}
QToolButton:pressed
{
	background-color:rgb(170, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="head_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
border-radius:35px;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mid" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>151</y>
      <width>389</width>
      <height>53</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>388</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <widget class="QtMaterialTabs" name="navigate" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>53</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>388</width>
       <height>53</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
border:none;
background-color:white;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="view_stack">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>204</y>
      <width>388</width>
      <height>468</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>468</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="message_page">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="message_list">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="relation_page">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="relation_navigate" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>51</height>
         </size>
        </property>
        <widget class="QtMaterialFlatButton" name="friend_btn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>194</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>好友</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QtMaterialFlatButton" name="group_btn">
         <property name="geometry">
          <rect>
           <x>194</x>
           <y>0</y>
           <width>194</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>群聊</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="relation_stack">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>420</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="friend_page">
         <widget class="QListView" name="friend_list">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>388</width>
            <height>417</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>480</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="group_page">
         <widget class="QListView" name="group_list">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>388</width>
            <height>417</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="dynamic_page">
     <widget class="QListView" name="dynamic_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>388</width>
        <height>468</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="bottom" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>672</y>
      <width>389</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <widget class="QToolButton" name="search_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="add_friend_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="dynamic_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="create_group_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialTabs</class>
   <extends>QWidget</extends>
   <header location="global">qtmaterialtabs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialflatbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
